packages/expo-modules-core/ios/AppDelegates/ExpoAppDelegate.swift

```diff
@@ -107,6 +122,8 @@ open class ExpoAppDelegate: UIResponder, UIApplicationDelegate {
     subscribers.forEach { $0.application?(application, didRegisterForRemoteNotificationsWithDeviceToken: deviceToken) }
   }
 
+
+
```

Comment:
Unfortunately integrating linters with Xcode is also very bad DX that we would like to avoid 😅 We also don't want to add any git hooks as it would slow down git for everyone working with the repo (and not all files are without issues still).

You can use
```
swiftlint lint --fix packages/expo-modules-core/ios/AppDelegates/ExpoAppDelegate.swift
```
to fix this particular issue. In the future @expo-bot will post review suggestions in its comments (for fixable rules) so I believe it would make it much easier to fix, even on a PR level 😉 

Another thing I noticed thanks to this PR is that the linter doesn't detect wrong indent size 😱 I'll need to check that out when we decide to upgrade SwiftLint on the CI. Some new rules came out in new versions that we'd like to add, so it will be a good occasion to revisit the config and maybe do some cleanups and fix all existing issues.